src/WizardsWardrobe.lua

```diff
@@ -21,6 +21,14 @@ local logger = LibDebugLogger( WW.name )
 WW.callbackManager = ZO_CallbackObject:New()
 
 
+function WW.IsAnySceneShown()
+  for sceneName, scene in pairs(SCENE_MANAGER.scenes) do
+	if scene:GetState() == SCENE_SHOWN and sceneName ~= "hud" and sceneName ~= "hudui" then

```

Comment:
Iterating potentially every scene before a setup swaps is not ideal. 
You could add your check to the [existing scene management](https://github.com/nicokimmel/wizardswardrobe/blob/20712710ea2b99ae8e31762c4720e9a5ef795691/src/WizardsWardrobeGui.lua#L117) or add a new event listener for scene changes to set a flag that can be quickly checked perhaps.